proxy/m2_actions.c

```diff
@@ -1357,11 +1379,13 @@ _m2_container_snapshot(struct req_args_s *args, struct json_object *jargs)
 
 	meta1_urlv_shift_addr(urlv);
 	CLIENT_CTX(ctx, args, type, 1);
+	char *url = _resolve_service_id(urlv[0]);
```

Comment:
That function seems to return a GLib-2.0 allocated string. Using a `gchar*` seems better.